src/style.css

```diff
@@ -1,3 +1,70 @@
+/* stylelint-disable no-descending-specificity */
```

Comment:
The use of stylelint-disable comments should be minimized. Instead of disabling the rules, it would be better to follow them to ensure good coding practices and consistency.